### DISCUSSION TITLE
I am lost and demotivated please help!!!

### DISCUSSION THREAD

**Discussion by BKM8383** [community/community#106306] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

HI everyone , I am a second year CSE student at a third tier private collage and i am lost , i lack motivation to keep growing at this point of time i just have so much to do , i have to learn 10 different things , i have to make a good profile on coding platform , i have to attend my classes , i have to study for my collage exams and assessments , i need some time for my own , I want to do some good courses and projects , i want to add some weight in my cv , i want to get some experience and i just cant focus on everything i do have created a time table but the time i am giving to activities is just not enough. I have no one to guide and to be honest don't have any friends either . Its not like i am week or a slow learner its just too much like currently i am learning JavaScript, java,(frontend)react, angular ,(backend)node.js , Mojang , sql ,ai models ,dsa , projects , linux , collage useless subjects and all together is really drains me out and when i look at people around me i regret it , i feel like i am too harm on myself. On the other hand when i see some of my class makes actually making real projects i feel like i am not that good . I have been doing leetcode but end up seeking answers from solution all because i don't want to lose my streak and i don't have time to learn those deep topics.

I am helpless and without any direction i am clueless where to go i want to go into software development but have 0 idea what actually to do and learn. and there is just looooot more to say ..........if someone can actually help me it would be great.

thank you.

**Comment by ZeeshanMukhtar1** [comment]
Hi @BKM8383 , I understand you're feeling overwhelmed with your workload and lack of direction. please brother Prioritize your tasks, set realistic goals, and focus on understanding core concepts deeply rather than trying to learn everything at once. Seek support from online communities, consider talking to professors or mentors, and remember to take breaks for self-care. It's okay not to have everything figured out; take one step at a time, and you'll find your way. if u feel any problem i am just a DM away..!

**Comment by mauriceoboya** [comment]
Hello! It sounds like you're dealing with a lot of pressure and feeling overwhelmed. It's not uncommon for students, especially in computer science, to feel this way with a demanding curriculum and the desire to excel in multiple areas.
Here are some suggestions to help you manage your situation
 Identify the most important tasks and focus on them first. Break down larger tasks into smaller, more manageable ones. This can make them less intimidating and help you stay organized. It's okay to say no to some commitments if you feel overwhelmed. Focus on what's essential for your academic and personal growth. Quality over quantity is important.  Understand your long-term goals and break them down into smaller, achievable milestones. This will give you a clearer sense of purpose and direction.

 while learning theoretical concepts is crucial, practical application is equally important. Try to work on small projects or apply what you've learned in real-world scenarios,  acknowledge and celebrate your achievements, no matter how small. This will help boost your confidence and motivation.
Remember, it's okay not to have everything figured out right away. Learning is a gradual process, and everyone progresses at their own pace. Take one step at a time, and don't be too hard on yourself.

**Comment by theburcu** [comment]
Hi there! Believe me when I say many people go through same things, you are not alone. However, if you are young and trying to achieve and complete things while your peers are living the best life (if this is the thing that bothers you, the feeling of missing life), try to focus on balancing things. Nothing is more important than your health and you should also enjoy your life.

1. Create a Schedule:
Develop a weekly schedule that includes your class times, study sessions, and other commitments.
Allocate specific time blocks for social activities, ensuring you have designated breaks to relax and socialize.

2. Prioritize Academics:
Stay on top of your coursework and assignments to avoid last-minute stress.
Use effective study techniques and time management to make the most of your academic hours.

3. Set Realistic Goals:
Establish achievable academic and social goals for each week or month.
Break down larger tasks into smaller, more manageable steps to make progress more feasible.

4. Stay Organized:
Keep your study materials, notes, and schedule well-organized to reduce time wasted searching for things.
Use tools like planners, calendars, or apps to stay on top of deadlines and events.

5. Learn to Say No:
Understand your limits and avoid overcommitting to too many activities or responsibilities.
Politely decline additional commitments if you feel it will compromise your academic or social well-being.

6. Effective Time Management:
Prioritize tasks based on importance and deadlines.
Avoid procrastination by breaking down tasks and starting them early.

7. Combine Activities:
Look for ways to combine social activities with academic responsibilities. For example, form study groups or participate in group projects with friends.

8. Take Breaks:
Allow yourself short breaks during study sessions to refresh your mind.
Use breaks to engage in social activities or hobbies to prevent burnout.

9. Stay Healthy:
Ensure you get enough sleep, eat well, and exercise regularly. A healthy lifestyle positively impacts both academic and social aspects of your life.

10. Communicate Effectively:
Keep open lines of communication with friends, family, and classmates about your commitments and schedule.
Discuss and negotiate shared responsibilities within your social circles.

11. Reflect and Adjust:
Regularly assess how well you're balancing your academic and social life.
Adjust your schedule and priorities as needed based on your experiences and evolving responsibilities.

Maybe these steps can help. If not, there are always other options like seeking professional help. ☺️
